packages/connect/src/data/connectSettings.ts

```diff
@@ -36,6 +36,9 @@ const parseManifest = (manifest?: Manifest) => {
     if (!manifest) return;
     if (typeof manifest.email !== 'string') return;
     if (typeof manifest.appUrl !== 'string') return;
+    // todo [connect10]: appName should become required
+    if (typeof manifest.appName !== 'undefined' && typeof manifest.appUrl !== 'string') return;
```

Comment:
Bit strange condition, it will never happen since `typeof manifest.appUrl !== 'string'` is handled above

Reply:
its a bug, should be 
```
if (typeof manifest.appName !== 'undefined' && typeof manifest.appName !== 'string') return;
```

Reply:
391b09103212951e50269373803d6abd39c3214a